src/Environment.jl

```diff
@@ -0,0 +1,29 @@
+
+const _NREFS = Ref(0)
+
+function Init(parts::MPIData;p4est_verbosity_level=P4est_wrapper.SC_LP_DEFAULT)
+  if !MPI.Initialized()
```

Comment:
do we really need this? 

At this point, you have been able to create `parts` (as you pass it as an argument). Thus, `MPI` should have been initialized at this point. 

I guess that we need here to check that MPI is Initialized, and raise and error otherwise?

Reply:
Done.